### PR TITLE
fix: make sure window has mounted before initilizing Headway

### PR DESCRIPTION
**Pull request by nathaliabj**
### Pre requirements:
- [ ] I have followed (at least) the [PR section of the contributing guide](https://github.com/lightdash/lightdash/blob/main/.github/CONTRIBUTING.md#sending-a-pull-request).

### Reference:
Closes: #1812 

### Description:
This PR makes sure window has loaded before initialising Headway

### Scope of the changes (select all that apply):
- [x] Frontend  
- [ ] Backend  
- [ ] Common  
- [ ] E2E tests  
- [ ] Docs  
- [ ] CI/CD

### CODE REVIEW

File: packages/frontend/src/components/NavBar/HelpMenu/index.tsx [hunk 1]
```diff
@@ -26,42 +26,55 @@ const HelpMenu: FC = () => {
...
     const { projectUuid } = useParams<{ projectUuid: string }>();
+    const [hasMounted, setHasMounted] = useState<boolean>(false);
 
-    const trackNotifications = {
-        user_id: user.data?.userUuid,
-        project_id: projectUuid,
-        organization_id: user.data?.organizationUuid,
-    };
+    useEffect(() => {
+        setHasMounted(true);
```
Comment (rephus) [inline packages/frontend/src/components/NavBar/HelpMenu/index.tsx:32]: is this not practically the same as the other useEffect() ?, just delaying a bit more ? I don't understand why this is fixing the issue..

**Review by rephus — CHANGES_REQUESTED** [review]
I can still hit this error if I block the URL or set the internet speed to Slow. This causes the whole site to crash . 
image

Maybe we could just check if `window.Headway `  is not undefined before doing `.init` ?

File: packages/frontend/src/components/NavBar/HelpMenu/index.tsx (hunk 1 above)
Comment (nathaliabj) [reply packages/frontend/src/components/NavBar/HelpMenu/index.tsx:32]: @rephus  i tried that however according to TS Headway just never will exist on window

Comment (rephus) [reply packages/frontend/src/components/NavBar/HelpMenu/index.tsx:32]: I see, well I still got that error on `render` so this does not fix that issue completely

**Pull request merged**